metron-interface/metron-alerts/src/app/pcap/service/pcap.service.ts

```diff
@@ -47,47 +48,47 @@ export class PcapService {
   public submitRequest(
     pcapRequest: PcapRequest
   ): Observable<PcapStatusResponse> {
-    return this.http.post('/api/v1/pcap/fixed', pcapRequest).pipe(
+    return this.http.post(this.appConfigService.getApiRoot() + '/pcap/fixed', pcapRequest).pipe(
       map(HttpUtil.extractData),
       catchError(HttpUtil.handleError)
     );
   }
 
   public getStatus(id: string): Observable<PcapStatusResponse> {
-    return this.http.get(`/api/v1/pcap/${id}`).pipe(
+    return this.http.get(this.appConfigService.getApiRoot() + `/pcap/${id}`).pipe(
       map(HttpUtil.extractData),
       catchError(HttpUtil.handleError)
     );
   }
 
   public getRunningJob(): Observable<PcapStatusResponse[]> {
-    return this.http.get(`/api/v1/pcap?state=RUNNING`).pipe(
+    return this.http.get(this.appConfigService.getApiRoot() + `/pcap?state=RUNNING`).pipe(
```

Comment:
I think since you're not using multi-line strings and string interpolation features here, it's better to use single quotes rather than a template literal.